src/docs/grid-intensity-cli/tutorials/getting-grid-intensity.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Getting grid intensity for a region
+description: In this tutorial you will use the Grid Intensity CLI to get both historical and near real-time data for a region.
+eleventyNavigation:
+  key: getting-grid-intensity
+  title: Getting grid intensity for a region
+  # parent: overview
+  sectionTitle: Tutorials
+  order: 10
+---
+# {{ title }}
+
+## Overview
+
+Knowing the grid intensity for the regions in which your code runs allows you to make better informed decisions about where and/or when to run it.
+
+In this tutorial, you will use the Grid Intensity CLI to:
+
+1. Get historical intensity data for a region.
+1. Change data providers, and
+1. Get near 0real-time grid intensity data for a region.
```

Comment:
tiny typo here

Reply:
Fixed in https://github.com/thegreenwebfoundation/developer-docs/pull/6/commits/b7567fc2fd5a05a983acd5df116ca058f5f77393